### DISCUSSION TITLE
v3.11.0

### DISCUSSION THREAD

**Discussion by lukevella** [lukevella/rallly#1386] [Announcements]
## What's Changed

* ✨ Use profile image from oauth provider by @lukevella in https://github.com/lukevella/rallly/pull/1330
* ✨ Use avatar on login page by @lukevella in https://github.com/lukevella/rallly/pull/1336
* 🐛 Fix empty session when logging out by @lukevella in https://github.com/lukevella/rallly/pull/1345
* 🔨 Use node:20-slim as runner by @xavier-calland in https://github.com/lukevella/rallly/pull/1350
* 💄 Update avatar colors by @lukevella in https://github.com/lukevella/rallly/pull/1351
* 🐛 Fix escaped values in emails by @lukevella in https://github.com/lukevella/rallly/pull/1368
* 🐛 fix ics download button by @a-mnich in https://github.com/lukevella/rallly/pull/1381
* 🐛 Fix name change by @lukevella in https://github.com/lukevella/rallly/pull/1384

## New Contributors
* @xavier-calland made their first contribution in https://github.com/lukevella/rallly/pull/1350
* @a-mnich made their first contribution in https://github.com/lukevella/rallly/pull/1381

**Full Changelog**: https://github.com/lukevella/rallly/compare/v3.10.0...v3.11.0

<hr /><em>This discussion was created from the release <a href='https://github.com/lukevella/rallly/releases/tag/v3.11.0'>v3.11.0</a>.</em>

**Comment by DoraDeu** [comment]
I just want to let you know how positive rallly developed over the last month and how much I appreciate the ongoing work of all contributors.

**Reply by lukevella** [reply]
Thanks @DoraDeu :)
